<span style="color: blue;">Deportes</span>. En un gimnasio sólo para mujeres se realizan ejercicios aeróbicos y ejercicios en aparatos. El circuito está constituido por nueve aparatos tonificadores y ocho estaciones para los ejercicios aeróbicos. En un día cualquiera, en hora pico (entre las 19 y 20 horas), llegan en promedio siete mujeres. En ese horario:

a) ¿Cuál es la probabilidad de que lleguen menos de cinco mujeres?  
b) ¿Cuál es la probabilidad de que estén ocupados todos los aparatos?  
c) Y en un día extraordinario, ¿cuál es la probabilidad de que tanto los aparatos como las estaciones estén ocupados?  
d) ¿Por qué es importante practicar algún deporte? ¿Qué beneficios trae su práctica? ¿Haces algún deporte? ¿A qué conclusiones llegas? Realiza una breve exposición con tus compañeros  



## a) probabilidad de que lleguen menos de cinco mujeres

La expresión “menos de cinco personas” significa:

$$P(X<5)$$

Utilizamos la regla del complemento:

$$\begin{align*}  P(X < 5) &= P(X \le 4) \\  &= P(X=0) + P(X=1) + P(X=2) + P(X=3) + P(X=4) \\  &= \frac{e^{-7}(7)^0}{0!} + \frac{e^{-7}(7)^1}{1!} + \frac{e^{-7}(7)^2}{2!} + \frac{e^{-7}(7)^3}{3!} + \frac{e^{-7}(7)^4}{4!} \\  &= 0.000912 + 0.006383 + 0.022341 + 0.052129 + 0.091226 \\  &= 0.172992  \end{align*}$$

por lo tanto:

$$P(X<5) = 0.172992$$

la probabilidad de que lleguen menos de cinco mujeres es aproximadamente 
$17.29\%$.

In [2]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import poisson
from IPython.display import HTML, display

# Parámetro de la distribución
lam = 7

# Valores que se mostrarán en la tabla y en la gráfica
x = np.arange(0, 7)

# Probabilidades P(X = x)
probabilidades = poisson.pmf(x, mu=lam)

# Construcción de la tabla HTML
html = """
<table style="
    border-collapse: collapse;
    width: 520px;
    text-align: center;
    font-family: Arial, sans-serif;
">
    <thead>
        <tr>
            <th style="
                border: 1px solid #000;
                padding: 10px;
                width: 200px;
                background-color: #f2f2f2;
            ">x (Mujeres que llegan)</th>
            <th style="
                border: 1px solid #000;
                padding: 10px;
                width: 320px;
                background-color: #f2f2f2;
            ">P(X = x)</th>
        </tr>
    </thead>
    <tbody>
"""

for valor, probabilidad in zip(x, probabilidades):
    html += f"""
        <tr>
            <td style="border: 1px solid #000; padding: 8px;">
                {valor}
            </td>
            <td style="border: 1px solid #000; padding: 8px;">
                {probabilidad:.6f}
            </td>
        </tr>
    """

html += """
    </tbody>
</table>
"""

display(HTML(html))

x (Mujeres que llegan),P(X = x)
0,0.000912
1,0.006383
2,0.022341
3,0.052129
4,0.091226
5,0.127717
6,0.149003


## b) probabilidad de que estén ocupados todos los aparatos

Asumiendo que las usuarias prefieren ocupar primero los 9 aparatos tonificadores disponibles, requerimos que lleguen exactamente 9 mujeres o mas (X >= 9) = 1 - P(X <= 8). El desglose matemático es:

$$\begin{align*} P(X \le 8) &= P(X=0) + P(X=1) + \dots + P(X=8) \\ &= 0.000912 + 0.006383 + 0.022341 + 0.052129 + 0.091226 + 0.127717 + 0.149003 + 0.149003 + 0.130377 \\ &= 0.729091 \end{align*}$$

por lo tanto:

$$P(X<8) = 27.09$$


## c) En un día extraordinario, ¿cuál es la probabilidad de que tanto los aparatos como las estaciones estén ocupados?

Para que todo el gimnasio esté lleno a su máxima capacidad (9  aparatos + 8 estaciones = 17  lugares), necesitamos que lleguen 17 o más mujeres ($X\ge17$).

$$\begin{align*}  P(X \geq 17) &= 1 - P(X \leq 16) \\  &= 1 - \left[ P(X=0) + P(X=1) + \dots + P(X=16) \right] \\  &= 1 - \left[ \frac{e^{-7}(7)^0}{0!} + \frac{e^{-7}(7)^1}{1!} + \dots + \frac{e^{-7}(7)^{16}}{16!} \right] \\  &= 1 - 0.999042 \\  &= 0.000958\end{align*}$$

## d)¿Por qué es importante practicar algún deporte?

Es importante porque ayuda con la salud a desestresarte haciendo que tu cuerpo genere dopamina y mejorando tu estado de animo

## ¿Qué beneficios trae su práctica? 

Ayuda a desestrezar liberando dopamina y mejorando el humor, ayuda a regular tu peso corporal

## ¿Haces algún deporte? 

Si salgo en bicicleta

## ¿A qué conclusiones llegas?

La practica de deportes nos ayuda a tener una mente sana y enfocar para superar retos 
